cmd/sriovdp/manager.go

```diff
@@ -100,13 +150,85 @@ func (rm *resourceManager) readConfig() error {
 	return nil
 }
 
-func (rm *resourceManager) initServers() error {
+// StoreConfig saves SR-IOV Device Plugin config into the file
+func (rm *ResourceManager) StoreConfig() error {
+	filename := filepath.Base(rm.configFile)
+	source, err := os.Open(rm.configFile)
+	if err != nil {
+		glog.Errorf("storeConfig(): error creating reading config file: %v", err)
+		return err
+	}
+	defer source.Close()
+
+	destination, err := os.Create(filepath.Join(rm.hostConfigDir, filename))
+	if err != nil {
+		glog.Errorf("storeConfig(): error creating creating config file on host: %v", err)
+		return err
+	}
+	defer destination.Close()
+
+	_, err = io.Copy(destination, source)
+	if err != nil {
+		glog.Errorf("storeConfig(): error during copy config file to host: %v", err)
+		return err
+	}
+	return nil
+}
+
+// CreateCdiSpec creates CDI definition for container runtime
+func (rm *ResourceManager) CreateCdiSpec() error {
```

Comment:
This is not used, please remove

Reply:
It's used at https://github.com/k8snetworkplumbingwg/sriov-network-device-plugin/pull/489/files#diff-f1a4e99ad0d1a580dda8274ab56d83450f41088bae0f4d0df0808d03c851d20dR216 and  https://github.com/k8snetworkplumbingwg/sriov-network-device-plugin/pull/489/files#diff-f1a4e99ad0d1a580dda8274ab56d83450f41088bae0f4d0df0808d03c851d20dR264